apps/src/applab/commands.js

```diff
@@ -1839,6 +1839,12 @@ applabCommands.analogRead = function (opts) {
   return Applab.makerlabController.analogRead(opts.pin, opts.callback);
 };
 
+applabCommands.onBoardEvent = function (opts) {
```

Comment:
TODO: (maybe as separate commit): pull out into makerlab for easier integration with other app types